sample/src/main/java/com/willowtreeapps/signinwithapple/sample/SampleActivity.kt

```diff
@@ -40,21 +33,28 @@ class SampleActivity : AppCompatActivity() {
         val service = SignInWithAppleService(
             clientId = "com.your.client.id.here",
             redirectUri = "https://your-redirect-uri.com/callback",
-            scope = "email name",
-            callback = signInCallback
+            scope = "email name"
         )
 
-        signInWithAppleButtonBlack.service = service
-        signInWithAppleButtonWhite.service = service
-        signInWithAppleButtonWhiteOutline.service = service
+        val client: SignInWithAppleClient = this
+
+        signInWithAppleButtonBlack.configure(service, client)
+        signInWithAppleButtonWhite.configure(service, client)
+        signInWithAppleButtonWhiteOutline.configure(service, client)
     }
 
-    override fun onStop() {
-        super.onStop()
+    // SignInWithAppleClient
+
+    override fun getFragmentManagerForSignInWithApple(): FragmentManager {
```

Comment:
Fine either way, but might be more idiomatic to single-line this
`override fun getFragmentManagerForSignInWithApple() = supportFragmentManager`

Reply:
I agree, but I wanted it to be really explicit as a sample code reference.